packages/@lwc/ssr-compiler/src/compile-js/index.ts

```diff
@@ -205,6 +213,32 @@ const visitors: Visitors = {
     },
 };
 
+function validateDecorators(decorators: EsDecorator[]) {
```

Comment:
```suggestion
function validateUniqueDecorator(decorators: EsDecorator[]) {
```
nit: say which validation is being done